src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -342,6 +378,38 @@ public function flush()
         return true;
     }
 
+    /**
+     * Remove all items from the cache.
+     *
+     * @param  string  $key
+     * @return bool
+     */
+    protected function forgetMany(array $keys)
+    {
+        $this->table()->whereIn('key', array_map(function ($key) {
+            return $this->prefix.$key;
+        }, $keys))->delete();
+
+        return true;
+    }
+
+    /**
+     * Remove all expired items from the given set from the cache.
+     *
+     * @return bool
+     */
+    protected function forgetManyIfExpired(array $keys, bool $prefixed = false)
```

Comment:
/**
     * Remove all expired items from the given set from the cache.
     */
    protected function forgetManyIfExpired(array $keys, bool $prefixed = false): bool

Reply:
Done. Thanks!